projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -182,7 +184,8 @@ public static function is_next_steps_tutorial_enabled() {
 	 * @param string $version The version of the asset file to get.
 	 */
 	public function enqueue_script( $variant, $dependencies, $version ) {
-		$script_dependencies = $dependencies ?? array();
+		$script_dependencies   = $dependencies ?? array();
+		$is_menu_panel_enabled = $this->is_menu_panel_enabled();
```

Comment:
[I learned recently](https://github.com/Automattic/jetpack/pull/45718/) that issuing blocking network requests before rendering the page kills performance (~20% slowdown!), so we should be a bit careful about this.

Let's move this call to the place it's needed only. We only need this on the backend in the connected wp-admin bar varaint. And in the case of the Gutenberg variant, I think we should fetch the experiment on the frontend.